phpunit.xml.dist

```diff
@@ -9,7 +9,7 @@
          processIsolation="false"
          stopOnFailure="false"
          syntaxCheck="false"
-         bootstrap="tests/bootstrap.php"
+         bootstrap="tests/tests/bootstrap.php"
```

Comment:
Only the fixtures might need to be autoloadable… tests that should be used outside the bundle should not be in tests, they should be in `src/Test`… but anyway, I thought about it more, it's not a BC break, b/c this is autoload-dev